pkg/ip/ip.go

```diff
@@ -714,3 +714,31 @@ func partitionCIDR(targetCIDR net.IPNet, excludeCIDR net.IPNet) ([]*net.IPNet, [
 
 	return left, excludeList, right
 }
+
+// KeepUniqueIPs transforms the provided multiset of IPs into a single set,
+// lexicographically sorted via a byte-wise comparison of the IP slices (i.e.
+// IPv4 addresses show up before IPv6).
+// The slice is manipulated in-place destructively.
+//
+// 1- Sort the slice by comparing the IPs as bytes
+// 2- For every unseen unique IP in the sorted slice, move it to the start of
```

Comment:
oops, I just realised that this sentence was always wrong! If you end up touching the PR again, and have time, could you replace `move it to the start of` with `move it to the end of`? I can also do it later, no problem.